releasenotes/notes/oracle-query-metrics-time-limit-0e7fe39d80ba885d.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Impose the time limit on query metrics processing. After exceeding the default limit of 20s, the agent stops emitting execution plans and fqt events.
```

Should "fqt" be capitalized?  What does it stand for?

```suggestion
    Impose a time limit on query metrics processing. After exceeding the default limit of 20s, the Agent stops emitting execution plans and fqt events.
```